pages/common/git-show-index.md

```diff
@@ -0,0 +1,12 @@
+# git show-index
+
+> Show packed archive index.
+> More information: <https://git-scm.com/docs/git-show-index>.
+
+- Read the '.idx' file for a Git packfile and dump its contents to stdout:
+
+`git show-index {{path/to/file}}`
+
+- Specify the hash algorithm for the index file (experimental):
+
+`git show-index --object-format={{hash_algorithm}} {{path/to/file}}`
```

Comment:
Since I'm assuming there's a limited number of supported hash algorithms here, I'd suggest it would be more helpful to list them here explicitly with our `{{something|another_thing|blah|...}}`/`{{something|another_thing|blah}}` syntax.

Reply:
That would be 
```suggestion
`git show-index --object-format={{sha1|sha256}} {{path/to/file}}`
```
I'm not sure which to put as the default tough.
>  The default is the algorithm for the current repository (set by `extensions.objectFormat`), or `sha1` if no value is set or outside a repository.

Reply:
The object format flag is also flagged as experimental, should we include that?

Reply:
Seems fine to me to include.

Reply:
It's almost 01:00 over here, I'll handle this tomorrow